src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -820,7 +814,9 @@ export default {
 		 * @param {string} htmlOrText the html content (or raw text with @mentions)
 		 */
 		updateValue(htmlOrText) {
-			const text = this.parseContent(htmlOrText)
+			let text = this.parseContent(htmlOrText)
+			// Some browsers add a newline at the end of the contenteditable
+			text = text === '\n' ? '' : text
```

Comment:
The comment seems to not document the following line. Let's adjust to something like

```suggestion
			// Browsers keep <br> after erasing contenteditable
			text = text === '\n' ? '' : text
```

- Some browsers => browsers (it is not browser-specific. Or is it?)
- At the end => after erasing (not the ending but entire content is checked)